packages/sdk/src/lib/dataProtectorSharing/getRentals.ts

```diff
@@ -0,0 +1,40 @@
+import { WorkflowError } from '../../utils/errors.js';
+import {
+  addressSchema,
+  booleanSchema,
+  throwIfMissing,
+} from '../../utils/validators.js';
+import type { GetRentalsParams, GetRentalsResponse } from '../types/index.js';
+import { SubgraphConsumer } from '../types/internalTypes.js';
+import { getRentalsQuery } from './subgraph/getRentalsQuery.js';
+
+export async function getRentals({
+  graphQLClient = throwIfMissing(),
+  renterAddress,
+  protectedDataAddress,
+  includePastRentals = false,
+}: SubgraphConsumer & GetRentalsParams): Promise<GetRentalsResponse> {
+  const vRenterAddress = addressSchema()
+    .label('renterAddress')
+    .validateSync(renterAddress);
+
+  // could accept ENS but should take iExec in args
+  const vProtectedDataAddress = addressSchema()
+    .label('protectedDataAddress')
+    .validateSync(protectedDataAddress);
+
+  const vIncludePastRentals = booleanSchema()
+    .label('includePastRentals')
+    .validateSync(includePastRentals);
+
+  try {
+    return await getRentalsQuery({
+      graphQLClient,
+      renterAddress: vRenterAddress,
+      protectedDataAddress: vProtectedDataAddress,
+      includePastRentals: vIncludePastRentals,
+    });
```

Comment:
No transformation here, we directly return what the subgraph gave us.

Previously:
 - Renaming `endDate` to `endDateTimestamp`
 - Renaming `duration` to `durationInSeconds`
 - Renaming `price` to `priceInNRLC`

These "transformations" costs us, for example we need to have two different types, almost the same but not entirely.

And I feel like we could name these fields the way we want directly in the subgraph schema, so these "transformations" feel weird here.

Reply:
yeah i agree. We should make a PR before the release to modify subgraph field naming so